app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -236,6 +246,7 @@ const SendAdaPage = ({
       className={`input ${isModal ? '' : 'send-address'} fullwidth`}
       name="send-address"
       placeholder="Receiving address"
+      data-cy="Address text field"
```

Comment:
Would not it be in general better to avoid white spaces in the `data-cy` values and instead use `AddressTextField` style everywhere? It is easier to introduce two white spaces instead of one and then you are surprised why it does not work.

Reply:
done